core/src/main/scala/kafka/log/LogCleaner.scala

```diff
@@ -167,8 +167,20 @@ class LogCleaner(initialConfig: CleanerConfig,
    */
   def shutdown(): Unit = {
     info("Shutting down the log cleaner.")
-    cleaners.foreach(_.shutdown())
-    cleaners.clear()
+    try {
+      cleaners.foreach(_.shutdown())
+      cleaners.clear()
+    } finally {
+      remoteMetrics()
+    }
+  }
+
+  def remoteMetrics(): Unit = {
```

Comment:
Should this method name be `removeMetrics` (a `v` instead of a `t`)?

Reply:
Thanks for catching. This is fixed.